test/behavior/basic.zig

```diff
@@ -572,8 +572,6 @@ test "comptime cast fn to ptr" {
 }
 
 test "equality compare fn ptrs" {
-    if (builtin.zig_backend == .stage2_spirv64) return error.SkipZigTest; // Uses function pointers
```

Comment:
This should also be flaky. The update script should respect tests which have `// flaky` (please add that here too)